EarlGrey/Action/GREYActions.m

```diff
@@ -161,18 +165,8 @@ @implementation GREYActions
     NSString *textStr;
     // If we're dealing with a text field in a web view, we need to use JS to get the text value.
```

Comment:
We can remove this comment as well or update it to indicate we are using JS to *clear* it.